test/integration/gardenlet/managedseed/managedseed_test.go

```diff
@@ -180,7 +180,11 @@ var _ = Describe("ManagedSeed controller test", func() {
 	})
 
 	JustBeforeEach(func() {
-		By("creating SecretBinding")
+		Eventually(func(g Gomega) {
+			g.Expect(mgrClient.Get(ctx, client.ObjectKeyFromObject(gardenNamespace), &corev1.Namespace{})).To(Succeed())
+		}).Should(Succeed())
```

Comment:
You use gardenNamespace in `BeforeEach`, so this doesn't help. You can move this to `BeforeSuite` right after the create call.